packages/serverpod/lib/src/server/log_manager/log_writer.dart

```diff
@@ -219,9 +219,17 @@ class SessionLogEntryCache {
   /// Name of streaming message currently being processed.
   String? currentEndpoint;
 
+  int _currentLogOrderId = 0;
+
   /// This is used internally by Serverpod to ensure the ordering of log entries
   /// and log queries are correct.
-  int currentLogOrderId = 0;
+  int get currentLogOrderId => _currentLogOrderId;
+
+  /// This is used internally by Serverpod to ensure the ordering of log entries
+  int get nextLogOrderId {
```

Comment:
Suggestion: rename to `createLogOrderId`.

If we inspect the places where this is used it makes little sense that it is called `nextLogOrderId`, maybe just calling it `createLogOrderId `makes that a bit more clear?